config/routes.rb

```diff
@@ -9,5 +9,7 @@
   resources :institutions do
     resources :harvest_sources, shallow: true
   end
+  resources :admin, only: [:index]
```

Comment:
I'm wondering if the root route that gets setup by the Krikri generator should point to `admin#index`. Thoughts?